controllers/gce/instances/utils.go

```diff
@@ -0,0 +1,13 @@
+package instances
+
+import (
+	compute "google.golang.org/api/compute/v1"
+
+	"k8s.io/ingress/controllers/gce/utils"
+)
+
+// Helper method to create instance groups.
+// This method exists to ensure that we are using the same logic at all places.
+func CreateInstanceGroups(nodePool NodePool, namer *utils.Namer, port int64) ([]*compute.InstanceGroup, *compute.NamedPort, error) {
```

Comment:
This just wraps around the namer, right? 
IIUC, the namer is actually the same one. 
Is it really worthy?

Reply:
This is to ensure that the same logic is used at both places.
It is to prevent scenarios where one place is updated but not the other.

Reply:
Do you still need this?

you only need to expose it at ClusterManager, right?

Reply:
Yes still need it. Its also called from backends.go

Reply:
Why not just remove this and do `nodePool.AddInstanceGroup(namer.IGName(), port)` in both places?

Reply:
Doing that risks someone changing the logic at one place without changing the other.
This is a simple wrapper method to ensure that the same code is used at both places.